src/typography/typography.css

```
@@ -15,6 +15,15 @@ body {
   text-decoration: underline;
 }
 
+.omrs-title-1 {
+  font-style: normal;
+  font-weight: bold; /* should we be using a variable here to .omrs-bold */
+  font-size: 2.125rem;
+  line-height: 113.8%;
+  letter-spacing: 0.023rem;
+  color: #000000;
```

We should be using the text-black css variable here:

```suggestion
  color: var(--omrs-text-black);
```